easybuild/easyconfigs/l/LHAPDF/LHAPDF-6.5.3-gompi-2022a.eb

```
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'LHAPDF'
+version = '6.5.3'
+
+homepage = 'http://lhapdf.hepforge.org/'
+description = """
+Les Houches Parton Density Function unified library
+
+First you need to adjust by `export LHAPDF_DATA_PATH=/local/.../path/` to your.
+Then you can run `lhapdf update` followed by `lhapdf install your_pdf`.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://www.hepforge.org/archive/lhapdf/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['57435cd695e297065d53e69bd29090765c934936b6a975ff8c559766f2230359']
+
+dependencies = [
+  ('Python', '3.10.4'),
+  ('Boost', '1.79.0')
+]
+
+configopts = "--disable-static --enable-python"
+
+sanity_check_paths = {
+	'files': ["lib/libLHAPDF.so",'bin/lhapdf-config'],
+	'dirs': ["lib","share/LHAPDF"],
```

Replacing the tabs by spaces, making the quotes consistent, adding a space after each comma, using a variable for `.so`, and check for `bin/lhapdf` as well (no need to check for the `lib` dir, as we already check for a specific file in that dir):
```suggestion
    'files': ["bin/lhapdf", "bin/lhapdf-config", "lib/libLHAPDF.%s" % SHLIB_EXT],
    'dirs': ["share/LHAPDF"],
```